test/fuzz/main.cc

```diff
@@ -19,6 +19,10 @@
 #include "test/test_common/environment.h"
 #include "test/test_common/utility.h"
 
```

Comment:
`#include "absl/debugging/symbolize.h"`

Reply:
Finally was able to reproduce the build failure by building with --config=asan.  It is unclear to me why the build worked without --config=asan.